pallets/liquidate/src/lib.rs

```diff
@@ -291,13 +292,13 @@ pub mod module {
                     // if (collateral_total_value * liquidation_threshold)/(debt_total_value) < 1 ,execute liquidation
                     let mut processing = true;
                     let collateral_liquidation_threshold_value = classify_collaterals.iter().fold(
-                        Price::MIN,
+                        MIN_PRICE,
                         |acc,&(_,_,_,total_sum_price,liquidation_threshold)|
 							// acc + total_sum_price * liquidation_threshold
 							match total_sum_price
-								.checked_mul(liquidation_threshold)
-								.and_then(|r| r.checked_div(RATE_DECIMAL))
-								.and_then(|r| r.checked_add(acc))
+								.checked_mul(&liquidation_threshold)
+								.and_then(|r| r.checked_div(&FixedU128::from_inner(RATE_DECIMAL)))
```

Comment:
Should not use `RATE_DECIMAL` any more if using `Rate` or `Ratio`

Reply:
fixed